homework/index.js

```diff
@@ -19,20 +19,49 @@
   function createAndAppend(name, parent, options = {}) {
     const elem = document.createElement(name);
     parent.appendChild(elem);
-    Object.entries(options).forEach(([key, value]) => {
+    Object.entries(options).forEach(([key, content]) => {
       if (key === 'text') {
-        elem.textContent = value;
+        elem.textContent = content;
       } else {
-        elem.setAttribute(key, value);
+        elem.setAttribute(key, content);
       }
     });
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function changeDateFormat(date) {
+    const timeFormat = new Date(date);
+    return timeFormat.toLocaleString();
+  }
+
+  function sortRepos(a, b) {
+    const first = a.name.toLowerCase();
+    const second = b.name.toLowerCase();
+    const key = first < second ? -1 : first > second ? 1 : 0;
+    return key;
   }
 
+  function renderRepoDetails(repo, ul) {
+    function createTable(table, heading, content) {
```

Comment:
This function should stand on itself, i.e. avoid embedded function definition like this unless there is a good reason to embed it.